tests/e2e/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "googlesitekit-e2e",
+  "description": "E2E testing infrastructure for Google Site Kit",
+  "main": "script.js",
+  "private": true,
+  "devDependencies": {
+    "expect-puppeteer": "^4.4.0",
+    "jest-puppeteer": "^4.4.0"
+  },
```

Comment:
We need to define all dependencies that are used by e2e tests here, even if they are still used by other parts as well, in which case we shouldn't remove them from the main package.json.

Here is the full list of dependencies that should be here:

```
{
	"dependencies": {
		"@wordpress/babel-preset-default": "^4.20.0",
		"@wordpress/e2e-test-utils": "^4.16.1",
		"@wordpress/jest-console": "^3.10.0",
		"amphtml-validator": "^1.0.38",
		"babel-jest": "^26.6.3",
		"dockerode": "^4.0.2",
		"expect-puppeteer": "^4.4.0",
		"jest": "^26.6.3",
		"jest-puppeteer": "^4.4.0",
		"jsdom": "^16.7.0",
		"puppeteer": "^10.4.0"
	}
}
```

Reply:
We shouldn't need jsdom in E2E dependencies I don't think.

Reply:
@aaemnnosttv it is used here: https://github.com/google/site-kit-wp/blob/f4f7360dadf8ff5f093d85a274d5d158451cf272/tests/e2e/matchers/to-have-valid-amp-for-visitor.js#L23-L24

Reply:
Oh, right I forgot about that 😄  I stand corrected!